### PR TITLE
docs: update documentation for starting storybook

### PR DESCRIPTION
**Pull request by trebor**
📜 Documentation

- improve clarity of steps needed to run demo.
- add critical missing step.

### CODE REVIEW

**Review by williaster — APPROVED** [review]
Thanks for updating this 👍

File: packages/superset-ui-plugins-demo/README.md [hunk 1]
```diff
@@ -10,11 +10,14 @@ Storybook of `@superset-ui` packages. See it live at
...
 #### Run storybook
 
-To view the storybook locally, you should first run `yarn install && yarn run bootstrap` in the
-`@superset-ui` monorepo root directory, which will install all dependencies for this package and
-sym-link any `@superset-ui` packages to your local system.
-
-After that run `yarn run storybook` which will open up a dev server at http://localhost:9001.
+To view the storybook locally, you should:
+
+1. Clone the [superset-ui](https://github.com/apache-superset/superset-ui) monorepo.
```
Comment (kristw) [inline packages/superset-ui-plugins-demo/README.md:15]: You should not need `superset-ui` repo to run `superset-ui-plugins` demo.
Only step 3-6 should be enough.

Comment (kristw) [reply packages/superset-ui-plugins-demo/README.md:15]: I will merge this and take care of them.

**Comment by trebor** [comment]
@kristw thanks!

**Pull request merged**